demo_templates/amqp.yaml

```diff
@@ -0,0 +1,27 @@
+- key: test_amqp_1
+  expect:
+    amqp:
+      exchange: exchange_1
+      routing_key: key_in
+      queue: key_in
+  actions:
+    - publish_amqp:
+        exchange: exchange_1
+        routing_key: key_out
+        payload: >
+          {
+            "amqp": "OK",
+            "data": {}
+          }
+
+- key: test_amqp_2
+  expect:
+    amqp:
+      exchange: exchange_1
+      routing_key: key_in
+      queue: key_in
+  actions:
+    - publish_amqp:
+        exchange: exchange_1
+        routing_key: key_out
+        payload_from_file: './files/colors.json'
```

Comment:
lgtm

just one question.  which directory `.` refers to?  the working directory, or the directory which contains this yaml file?  i assume the latter.